lib/pubsub_client/null_subscriber.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module PubsubClient
+  # A null object to act as a subscriber when clients are in dev or test
+  class NullSubscriber
+    # This adds a subset of the available methods on the
+    # Google::Cloud::PubSub::ReceivedMessage, which is what
+    # gets yielded by the subscription when configuring the listener.
+    # For a list of methods, see the following link:
+    # https://googleapis.dev/ruby/google-cloud-pubsub/latest/Google/Cloud/PubSub/ReceivedMessage.html
+    NullResult = Struct.new(:acknowledge!) do
+      def data
+        '{"key":"value"}'
+      end
+
+      def ordering_key
```

Comment:
I believe the new property ingress is using ordered messages, so I am exposing that here.

Reply:
We are not using ordering key as of now. It needed additional changes to publisher which I tried locally but then we dropped the plan.